nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/Property.java

```diff
@@ -281,6 +281,14 @@
      */
     void setReadOnly(boolean value);
 
+    /**
+     * Returns if this property is flag as secured in system.
+     *
+     * @return if this property is flag as secured in system
+     * @since 11.1
+     */
+    boolean isSecured();
```

Comment:
@return {@code true} if this property is flag as secured in system, {@code false} otherwise

Reply:
Ideally just: "Returns {@code true} if the property is flagged as secured." and removing the @return line that doesn't add anything. But this is a detail no change needed.

Reply:
Agree, applied the change as I had to apply some other minor changes.